doc/source/examples/incompressible-flow/3d_nitsche_mixer_with_pbt_impeller/nitscher_mixer_with_pbt_impeller.rst

```diff
@@ -0,0 +1,279 @@
+
+For this configuration, the Reynolds number is defined as:
+
+  .. math::
+    Re = \frac{ND^2}{\nu}
+
+where :math:`N` is the angular velocity in Rotation Per Second (RPS) and :math:`\nu` is the kinematic viscosity. We investigate this case at a Reynolds number of 300 for the flow reaches a pseudo steady-state. However, this example can easily be extend to higher values of the Reynolds number by lowering the kinematic viscosity. It will be necessary to increase the mesh resolution for the solution to be meaningfully accurate.
+
+
+Parameter file
+--------------
+
+All the sections of the parameter file used in this case have been already explained in previous examples with the exception of the Nitsche immersed boundary. However, for the sake of completeness, the important sections are briefly explained.
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection simulation control
+      set method                  = bdf2
+      set output name             = mixer
+      set output path             = ./output/
+      set output frequency        = 2
+      set time end                = 5
+      set time step               = 0.02
+      set output boundaries       = true
+    end
+
+Since the flow generated by the agitator is naturally transient, we use a second-order ``bdf2`` time-stepping method. We simulate the flow for a total of 5 seconds. We set ``output boundaries = true`` to generate auxiliary visualisation file that will enable us to visualise the walls of the vessel without the inside cells and  thus facilitate the visualisation of our mixing set-up.
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+  subsection physical properties
+    subsection fluid 0
+      set kinematic viscosity            = 0.001
+    end
+  end
+
+The kinematic viscosity is set to :math:`0.001`. This, combined with the angular velocity of 1 RPS and the diameter of the impeller of 0.3, will allow us to simulate the vessel at :math:`Re=300`.
+
+
+Mesh
+~~~~~
+
+The ``mesh`` subsection specifies the computational grid:
+
+.. code-block:: text
+
+  subsection mesh
+      set type                 = dealii
+      set grid type            = subdivided_cylinder
+      set grid arguments       = 2:0.5:0.5
+      set initial refinement   = 3
+  end
+
+Because we use an immersed boundary approach, we only need to mesh the vessel without considering the impeller. This greatly simplifies the meshing process. The mesh can be generated straightforwardly using the deal.II mesh generator for a subdivided cylinder. We set the half-height and the radius of the vessel to be :math:`0.5`. This will generate a vessel in which the axis of the impeller will be along the :math:`x` coordinate. The bottom of the vessel will be on the :math:`x=-0.5` plane. This is not really intuitive, but this is a consequence of some of the inherent limitations of the mesh generators.
```

Comment:
I think it'd be good to refer to the documentation website here.

Reply:
I think it's already referred to a lot in the previous tutorials, but I'll add a reference if you think that makes sense.